README.md

```diff
@@ -341,6 +341,10 @@ cargo {
 }
 ```
 
+### generateBuildId
+
+Generate a build-id for the shared library during the link phase.
+
```

Comment:
Any suggestions on how to better describe this?  I think a link would be great, but when I did a quick search I didn't find a good page to send people to.

Reply:
I'm also open to changing the config name.

Reply:
I don't see a better link either; maybe https://linux.die.net/man/1/ld?  I see some options, but that can be for the next person to monkey with.